stdlib/LinearAlgebra/src/eigen.jl

```diff
@@ -378,6 +367,18 @@ julia> F.values
  0.0 - 1.0im
 
 julia> F.vectors
+2×2 Array{Complex{Float64},2}:
+  0.0-1.0im   0.0+1.0im
+ -1.0-0.0im  -1.0+0.0im
+
+julia> vals, vecs = F; # destructuring via iterationw
```

Comment:
`iterationw` -> `iteration` (no big deal, but since this is part of the user facing docs I think it is worth updating)